examples/langchain-rag-ec2/langchain_rag.py

```diff
@@ -163,3 +184,16 @@ def invoke(self, user_prompt: str):
     # method, and the module functions as a remote service.
     user_input = input("Ask a question about NYC tenants rights and responsibilities: ")
     print(rag_app.invoke(user_input))
+
+    # ## Standing up an endpoint
```

Comment:
don't we want this commented out too? or no

Reply:
oh yea we can comment out in the source file, but it will still show up as normal looking python code in the examples web page